certbot/tests/reporter_test.py

```diff
@@ -19,8 +20,10 @@ def setUp(self):
     def tearDown(self):
         sys.stdout = self.old_stdout
 
-    def test_multiline_message(self):
+    @mock.patch('certbot.reporter.os.getpid')
+    def test_multiline_message(self, mock_getpid):
         self.reporter.add_message("Line 1\nLine 2", self.reporter.LOW_PRIORITY)
+        mock_getpid.return_value = self.rep_module.INITIAL_PID
         self.reporter.atexit_print_messages()
```

Comment:
We can just use `self.reporter.print_messages()` here and undo all other changes to this test.